catalogue/webapp/services/catalogue/images.js

```diff
@@ -51,11 +60,15 @@ const imageIncludes = ['visuallySimilar'];
 
 export async function getImage({
   id,
-  env = 'prod',
+  toggles,
 }: GetImageProps): Promise<Image | CatalogueApiError> {
-  const url = `${rootUris[env]}/v2/images/${id}?include=${imageIncludes.join(
-    ','
-  )}`;
+  const apiOptions = globalApiOptions(toggles);
+  let url = `${
+    rootUris[apiOptions.env]
+  }/v2/images/${id}?include=${imageIncludes.join(',')}`;
+  if (apiOptions.indexOverrideSuffix) {
+    url += `&_index=images-${apiOptions.indexOverrideSuffix}`;
```

Comment:
Could we not mutate pretty please? Why not have a set of params and set them that way?

e.g.
```
type ImageApiOptions = { _index?: string }
```